.gitignore

```diff
@@ -50,3 +50,4 @@ ubuntu*.log
 
 # EEPROM image file (created by SITL)
 eeprom.bin
+/Debug/
```

Comment:
What is this for?

Reply:
It was generated by Eclipse. I forgot and commited it. I will fix it in the next push.

Reply:
I will fix it

Reply:
fixed